.travis.yml

```diff
@@ -33,6 +38,7 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/golang/protobuf/proto
   - go get -u github.com/golang/protobuf/protoc-gen-go
+  - go get -u google.golang.org/genproto/googleapis/rpc/...
```

Comment:
True, that's a left over artefact from fighting the `rpc.Status` thing -- removed.